File: go/1020.go
package main

import "fmt"

func main() {
	var d, a, m int
	fmt.Scanf("%d", &d)
	a = d / 365
	d = d % 365
	m = d / 30
	d = d % 30
	fmt.Printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", a, m, d)
}

File: lua/1020.lua
local d = tonumber(io.read())

local a = d // 365
d = d % 365

local m = d // 30
d = d % 30

print(string.format("%d ano(s)", a))
print(string.format("%d mes(es)", m))
print(string.format("%d dia(s)", d))
